Dockerfile-server

```diff
@@ -59,16 +59,16 @@ ENV GUNICORN_BIND="0.0.0.0:8000"
 ENV GUNICORN_TIMEOUT=400
 ENV GUNICORN_MAXREQUESTS=1200
 
-CMD gunicorn \
+CMD ["sh", "-c", "gunicorn \
```

Comment:
This is addressing a warning from the Dockerfile linter. Tested on stage and the server runs fine

Reply:
💯